src/pages/ProfilePage/ProfilePage.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import './ProfilePage.module.css';
+import styles from './ProfilePage.module.css'
+
+function ProfilePage() {
+    <img className={styles.image} src = 'tommy.png'></img>
```

Comment:
you didn't add tommy.png! as a result, it doesn't appear. just add back tommy.png into your PR (put it in the "public" folder and import it from there)

Reply:
<img width="649" alt="Screen Shot 2023-03-28 at 10 21 43 PM" src="https://user-images.githubusercontent.com/123126201/228434222-dc89a7f3-319a-4548-a476-434ce317b833.png">

I imported it and made sure that pic was in the public folder